loconfig.json

```diff
@@ -56,6 +56,14 @@
                 ],
                 "outfile": "{% paths.svgs.dest %}/sprite.svg"
             }
-        ]
+        ],
+        "purgeCSS": {
```

Comment:
We should find a better way to configure "sub-tasks" like Autoprefixer and PurgeCSS.

<details>
<summary>Example 1</summary>

```json
"styles": [
    {
        "infile": "{% paths.styles.src %}/critical.scss",
        "outfile": "{% paths.styles.dest %}/critical.css"
    },
    {
        "infile": "{% paths.styles.src %}/main.scss",
        "outfile": "{% paths.styles.dest %}/main.css",
        "options": {
            "purgecss": {
                "content": [
                    "./www/**/*.html",
                    "./views/app/template/**/*.mustache",
                    "./src/App/Template/*.php",
                    "./assets/scripts/**/*"
                ]
            }
        }
    }
]
```
</details>

<details>
<summary>Example 2</summary>

```json
"styles": {
    "entries": [
        {
            "infile": "{% paths.styles.src %}/critical.scss",
            "outfile": "{% paths.styles.dest %}/critical.css"
        },
        {
            "infile": "{% paths.styles.src %}/main.scss",
            "outfile": "{% paths.styles.dest %}/main.css",
        }
    ],
    "options": {
        "purgecss": {
            "content": [
                "./www/**/*.html",
                "./views/app/template/**/*.mustache",
                "./src/App/Template/*.php",
                "./assets/scripts/**/*"
            ]
        }
    }
}
```
</details>

Reply:
I think I like the option 2 better!